src/instruments/src/SD/Pages/Hyd/common.tsx

```diff
@@ -0,0 +1,313 @@
+            {
+                id: 'ptu2',
+                className: 'AmberLine',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu3',
+                className: 'AmberLine NoFill',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu4',
+                className: 'AmberLine',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu5',
+                className: 'Hide',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'triangle1',
+                className: '',
+                colour: 'Amber',
+                orientation: -90,
+                fill: 0,
+            },
+            {
+                id: 'triangle2',
+                className: '',
+                colour: 'Amber',
+                orientation: -90,
+                fill: 0,
+            },
+            {
+                id: 'triangle3',
+                className: '',
+                colour: 'Amber',
+                orientation: 90,
+                fill: 0,
+            },
+        ],
+    },
+    {
+        scenario: 'normal',
+        format: [
+            {
+                id: 'ptu1',
+                className: 'Hide',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu2',
+                className: 'GreenLine',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu3',
+                className: 'GreenLine NoFill',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu4',
+                className: 'GreenLine',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'ptu5',
+                className: 'GreenLine Hide',
+                colour: '',
+                orientation: 0,
+                fill: 0,
+            },
+            {
+                id: 'triangle1',
+                className: '',
+                colour: 'Green',
+                orientation: -90,
+                fill: 0,
+            },
+            {
+                id: 'triangle2',
+                className: '',
+                colour: 'Green',
+                orientation: -90,
+                fill: 0,
+            },
+            {
+                id: 'triangle3',
+                className: '',
+                colour: 'Green',
+                orientation: 90,
+                fill: 0,
+            },
+        ],
+    },
+    {
+        scenario: 'elec-pump',
```

Comment:
One more thing: I can't find any usage of the `elec-pump` scenario, if it's actually unused it should be removed here